### PR TITLE
119 136 137 update get started

### PR DESCRIPTION
**Pull request by thomasp-b**
### Thank you for your Pull Request! 

We have developed a Pull Request template to aid you and our reviewers.  Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the logrx codebase remains robust and consistent.  

### The spirit of logrx

While many packages to facilitate the logging of code already exist in the R ecosystem, it is hard to find a solution that works well for clinical programming applications. Many logging implementations are more implicit and rely on user input to create the log for the execution of a script. While this is useful for logging specific events of an application, in clinical programming a log has a set purpose.

logrx is built around the concept of creating a log for the execution of an R script that provides an overview of what happened as well as the environment that it happened in. We set out to create a flexible logging utility that could provide the necessary information to anyone reviewing the code execution so they can recreate the execution environment and run the code for themselves. Please make sure your Pull Request meets this **spirit of logrx**.

Please check off each taskbox as an acknowledgment that you completed the task. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `dev` branch until you have checked off each task.

- [x] The spirit of logrx is met in your Pull Request
- [x] Check that your Pull Request is targeting the dev branch, Pull Requests to master should use the [Release Pull Request Template](https://github.com/atorus-research/logrx/blob/main/.github/PULL_REQUEST_TEMPLATE/release.md)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x] Updated relevant unit tests or have written new unit tests
- [x] Creation/updates to relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Update NEWS.md if the changes pertain to a user-facing function (i.e. it has an @export tag) or documentation aimed at users (rather than developers)
- [x] Address any updates needed for vignettes and/or templates
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

### CODE REVIEW

File: vignettes/logrx.Rmd [hunk 1]
```diff
@@ -25,22 +25,34 @@ an R script which enables traceability and reproducibility of the executed code.
...
-* `metadata`: The logrx `package` information
-* `user and file information`: The user that generates the log, file name and path of the R script
+* `metadata`: logrx `package` information
+   + Version of the package
+   + Type of build
+   + Link to the GitHub repository
+* `user and file information`
+   + User that generated the log
+   + Name and path of the script for which the log was generated
 * `hash_sum`:  A unique sha1 is created for the log file
```
Comment (bms63) [inline vignettes/logrx.Rmd:35]:
```suggestion
* `hash_sum`:  A unique hashsum is created for the log file
```

File: vignettes/logrx.Rmd [hunk 2]
```diff
@@ -25,22 +25,34 @@ an R script which enables traceability and reproducibility of the executed code.
 # Log attributes
 The following attributes are recorded in the log:
 
-* `metadata`: The logrx `package` information
-* `user and file information`: The user that generates the log, file name and path of the R script
+* `metadata`: logrx `package` information
```
Comment (bms63) [inline vignettes/logrx.Rmd:28]:
```suggestion
* `metadata`: `logrx` package information
```

File: vignettes/logrx.Rmd (hunk 1 above)
Comment (bms63) [reply vignettes/logrx.Rmd:35]: The hash_sum should go under the `user and file info` section.  We also need a new log rendered for the screenshot that has the `File HashSum:` section in it

**Review by bms63 — CHANGES_REQUESTED** [review]
image

Can we get rid of that according ??

Can we put in a section for running with the addin as well?

**Comment by kodesiba** [comment]
@bms63 I wanted to check if you had other feedback.  If you're done then @thomasp-b if you have time to knock out the few changes left we can get this through

**Comment by thomasp-b** [comment]
> @bms63 I wanted to check if you had other feedback. If you're done then @thomasp-b if you have time to knock out the few changes left we can get this through

I'll work on what needs to be finalised tomorrow before our catch-up.

**Comment by kodesiba** [comment]
@thomasp-b checking in on the status of this, I can pick up the rest of the docs once this is done.

**Review by bms63 — APPROVED** [review]
LGTM!

**Comment by kodesiba** [comment]
@bms63 looking at the merge conflict and it looks like in #138 we removed the adsl.R file and I think added it to the tidylog examples.  Do you have time to reconcile these and make sure we have the whole data-raw folder updated so it's consistent with what we have in the dev branch?

**Pull request merged**